user_guide/src/multiple_files/intro.md

```diff
@@ -33,9 +33,9 @@ pl.scan_csv("my_many_files_*.csv").show_graph()
 ## Reading and processing in parallel
 
 If your files don't have to be in a single table you can also build a query plan for each file and execute them in paralllel
-on the polars thread pool.
+on the `Polars` thread pool.
 
-All query plans are executed embarassangly parallel and don't require any communication.
+All query plan execution is incredibly parallel and doesn't require any communication.
```

Comment:
"embarassangly parallel" is a technical term. It must remain in this sentence. It has more guarantees than "incredibly" which is has semantic meaning.

Reply:
TIL https://en.m.wikipedia.org/wiki/Embarrassingly_parallel#:~:text=In%20parallel%20computing%2C%20an%20embarrassingly,a%20number%20of%20parallel%20tasks.